sources/Outputs/include/Constants.h

```diff
@@ -37,27 +37,39 @@ hash(const std::string& str) {
   return std::hash<std::string>{}(str);
 }
 
+/**
+ * @brief Computes Qmax
+ *
+ * Pow = P^2 / sqrt(P^2 + Q^2) => Q = P * sqrt(1/pow^2 - 1)
```

Comment:
powerFactor instead of Pow.
Also use PMax instead of P and QMax instead of Q in the formula

Reply:
OK